src/app-logic/url-handling.js

```diff
@@ -922,6 +973,62 @@ const _upgraders = {
     const idleCategoryIndex = profile.meta.categories.findIndex(
       category => category.name === 'Idle'
     );
+    const otherCategoryIndex = profile.meta.categories.findIndex(
+      category => category.name === 'Other'
+    );
+
+    /**
+     * This function returns the stack index of the first occurrence of the given
+     * CallNodePath. Assumes the implementation filter of CallNodePath is 'js'.
+     */
+    function getStackIndexFromVersion5JSCallNodePath(
```

Comment:
This is a new upgrader, and should have its own commit. I don't think it's correct yet.